pkg/mgrconfig/load.go

```diff
@@ -94,12 +94,49 @@ func defaultValues() *Config {
 		Procs:          6,
 		PreserveCorpus: true,
 		Experimental: Experimental{
-			RemoteCover: true,
-			CoverEdges:  true,
+			RemoteCover:      true,
+			CoverEdges:       true,
+			DescriptionsMode: DescriptionsMode(ManualDescriptions).String(),
 		},
 	}
 }
 
+type DescriptionsMode int
+
+const (
+	ManualDescriptions = iota
+	AutoDescriptions
+	AnyDescriptions
+	Invalid
+)
+
+var (
+	enumToStr = map[DescriptionsMode]string{
```

Comment:
The name needs to include "descriptions", otherwise it's too generic (any enum can use this name).